08-Placing-Barriers/02-Fixed-particles.ipynb

```diff
@@ -0,0 +1,686 @@
+{
```

Comment:
<p>I think the name of this section would make more sense if it were called "Arbitrarily Shaped Barriers" or something like that.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/#comment-1392966260'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I agree with Tommy that the section title can have more information. But the title should imply arbitrary shapes of barriers are made of fixed particles to distinguish from next sections with smooth-surface wall functions.</p>

Reply:
I disagree. Future extensions to HOOMD might allow arbitrary mesh boundaries making "Arbitrarily Shaped Barriers" insufficient. "Creating Arbitrarily Shaped Barriers by Placing Fixed Particles" is far too long a title and contains redundant information given that it is in the context of the tutorial title "Placing barriers in the simulation box".

Keep in mind that the primary mode of navigation to tutorials will be in the documentation page sidebar:
<img width="244" alt="image" src="https://github.com/glotzerlab/hoomd-examples/assets/2197568/1a3ca5f6-2f8f-4a04-9859-39f1421240e1">
where short section titles are easier to read and are clearly placed next to the tutorial title to provide context.